### PR TITLE
ref task #7158 create layouts

### PR DESCRIPTION
**Pull request by hieunguyen061197**
## Redmine URL
 -https://edu-redmine.framgia.vn/issues/7158
screenshot from 2019-03-01 09-56-37

## Reproduce steps
 - for Bug only.

## What kind of page or function is need to check
 - 
 - 
 - 

## Before｜After
Before|After|
--|--|
Before content|After content|

## Check contents list
 - 

## Other
 - Test ENV
 - Batch, services,... need run for this pull.
screenshot from 2019-03-01 09-59-38

### CODE REVIEW

**Comment by hieunguyen061197** [comment]
ready

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
-end
+    def full_title page_title = ""
+      base_title = "TopMen"
```
Comment (yanghoxom) [inline app/helpers/application_helper.rb]: sao không dùng i18n hả chú

File: app/views/layouts/_footer.html.erb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<footer class="footer">
+  <small>
+    <b><%= t ".TopMen" %></b>
+    <%= t "by" %> <b>Hung |Hieu </b>
+  </small>
+  <nav>
+    <ul>
+      <li><%= link_to "0123456789", '#', class:"fa fa-phone-square" %></li>
```
Comment (yanghoxom) [inline app/views/layouts/_footer.html.erb]: mấy cái này đưa vào gem config đi

File: app/views/layouts/_rails_default.html.erb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<%= csrf_meta_tags %>
+    <%= stylesheet_link_tag    "application", media: "all",
```
Comment (yanghoxom) [inline app/views/layouts/_rails_default.html.erb]: fix indent đi em

**Comment by hieunguyen061197** [comment]
ready

**Pull request merged**